doc/tutorials/plugins.md

```diff
@@ -430,3 +431,9 @@ Plugin *ELEKTRA_PLUGIN_EXPORT(line)
 ```
 
 For further information see [the API documentation](https://doc.libelektra.org/api/current/html/group__plugin.html).
+
+## Note on direct method calls via external integrations
+
+Some applications want to call Elektra methods directly via native access (eg. [JNA](https://github.com/java-native-access/jna) for Java).
+A `KeySet` is a data structure over which functions can iterate. If the last element is reached, you have to explicitly call `rewind()`
```

Comment:
I do not think it matters if you reached the last element or not. As far as I know, you should always reset the pointer if you changed it (e.g. via a call of `ksNext`).